src/diffpy/utils/tools.py

```diff
@@ -30,31 +68,31 @@ def _sorted_merge(*dicts):
 
 def _create_global_config(args):
     username = input(f"Please enter the name of the user to put in the diffpy global config file "
-                     f"[{args.get("username", "")}]:  ").strip() or args.get("username", "")
+                     f"[{args.get('username', '')}]:  ").strip() or args.get("username", "")
```

Comment:
I changed the double quotes inside the f-string to single quotes to avoid syntax error